taf/testlib/virtual_env.py

```diff
@@ -343,16 +334,13 @@ def ensure_public_access(self, try_reuse=False, networks_client=None, routers_cl
         net_ip = netaddr.IPNetwork(_mgmt_ip_cidr)
```

Comment:
mgmt._ip_cidr and generally mgmt network is misleading here. It should be renamed to e.g.: external_gw_ip_cidr , external network.